scripts/create_backup_pd

```diff
@@ -82,7 +77,7 @@ echo '>>>>>>>>>>>>>>>>>>>> Loaded source code to the image.'
 
 sudo umount ${LOOP_DEVICE}
```

Comment:
I think we need to umount and disconnect loop device before tar the disk image, otherwise, I got the error below:
tar: disk.raw: file changed as we read it